code/game/turfs/walls/wall_types.dm

```diff
@@ -273,11 +273,24 @@
 //	icon_state = "title_holiday"
 	layer = FLY_LAYER
 	pixel_x = -64
+	//random title as starting point so it isnt choking every time you see it.
+	var/list/total_titles
```

Comment:
Why does this exist? This is completely pointless. Using an index var is much more efficient (like you had before)

Reply:
thats literally what you wanted.

Reply:
What do you mean it is completely pointless, it is required to get the amount of icons in the file instead of manually putting the number.